src/containers/sidebar.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import Divider from "@material-ui/core/Divider";
+import Drawer from "@material-ui/core/Drawer";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemIcon from "@material-ui/core/ListItemIcon";
+import ListItemText from "@material-ui/core/ListItemText";
+import IconButton from "@material-ui/core/IconButton";
+import ChevronLeftIcon from "@material-ui/icons/ChevronLeft";
+import InfoIcon from "@material-ui/icons/Info";
+import SchoolIcon from "@material-ui/icons/School";
+import AccountIcon from "@material-ui/icons/AccountCircle";
```

Comment:
should be `AccountCircle`?

Reply:
![Screenshot from 2019-03-28 19-07-58](https://user-images.githubusercontent.com/1000088/55153237-ec1b6380-518c-11e9-9fba-37d75d28efe2.png)

Doesn't seem to show any errors on my part, I'm just explicitly naming the default export so it's clear that its an icon.

Reply:
yep, all works fine. 

also here's a nitpick, you could reduce all your imports to 
```{Divider, Drawer, IconButton, List, ListItem, ListItemIcon, ListItemText, Typography} from "@material-ui/core";```
Great job on that sidebar component btw

Reply:
Thanks! I was planning to declare them that way but in the documentation of Material UI, it says that we should use that way of declaring if the environment supports "tree-shaking". I was not sure if our build system supports it so opted for the more verbose approach.

https://material-ui.com/style/icons/ (Imports section)

Reply:
Oh, I see. I've never heard of *tree shaking* before. It looks like a way of instructing webpack to eliminate any unused imports, and with our config, we're not doing that. But we can leave imports as they are just be safe, your call :)
in any case, I believe you can merge the PR

Reply:
@denibulkashvili We're using the latest `create-react-app`, and it uses webpack 4. So tree shaking should be enabled by default.

Reply:
@davidkwan95 thanks for the explanation